rrmngmnt/network.py

```diff
@@ -583,3 +585,38 @@ def if_down(self, nic):
         cmd = "ip link set down %s" % nic
         rc, _, _ = self.host.run_command(shlex.split(cmd))
         return not bool(rc)
+
+    def is_connective(self, ping_timeout=20.0, ipv6=False):
+        """
+        Check if host network is connective via ping command
+
+        :param ping_timeout: time to wait for response
+        :type ping_timeout: float
+        :param ipv6: if address IPV6
```

Comment:
I think this param is not necessary, you can use netaddr.valid_ipv6(self.host.ip) function to decide what command to use. like:

```
ping_cmd = "ping6" if netaddr.valid_ipv6(self.host.ip) else "ping"
```